### PR TITLE
Update Workspace invite documentation

### PR DESCRIPTION
**Pull request by jwitz**
Resolves https://github.com/astronomer/docs/issues/2534

### CODE REVIEW

File: astro/add-user.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ As a Workspace Admin or Organization Owner, you can add new team members to Astr
 ## Prerequisites
 
 - To add, edit, or remove Organization users, you need Organization Owner permissions.
-- To add edit, or remove Workspace users, you need Workspace Admin permissions for a given Workspace. The user must also already be a part of the Organization that hosts the Workspace.
+- To add edit, or remove Workspace users, you need Workspace Admin permissions for a given Workspace.
```
Comment (chomipi88) [inline astro/add-user.md:13]: FYI... the removed condition is still true for non-Org Owners.

File: astro/add-user.md [hunk 2]
```diff
@@ -53,7 +53,7 @@ See [User permissions](user-permissions.md) to view the permissions for each ava
    
 3. In the **Access Management** tab, click **Add Member**.
 
-4. Select the user's name and email address in the **Organization Member** list.
+4. Select the user's name and email address in the **Organization Member** list, or enter the email for a user hasn't yet been added to your Organization. If you invite a new user to a Workspace using their email, that user is also invited to the Organization as an Organization Member.
```
Comment (chomipi88) [inline astro/add-user.md:56]: Only possible for Org Owners

File: astro/release-notes.md [hunk 3]
```diff
@@ -78,6 +78,7 @@ See [Astro Hosted resource reference](resource-reference-hosted.md) for more inf
 ### Additional improvements
 
 - You can now add a new Astro user to Workspaces before the user has accepted their invite.
+- You can now add a user to a Workspace even if the user hasn't been added to your Organization. Users added to Workspaces this way are automatically added to your Organization as an Organization Member.
```
Comment (chomipi88) [inline astro/release-notes.md:81]: Only a capability for Org Owners. Only Org Owners have the permissions to add new users to an Org.

Comment (jwitz) [inline astro/release-notes.md:81]:
```suggestion
- If you have Organization Owner permissions,  you can now add a user to a Workspace even if the user hasn't been added to your Organization. Users added to Workspaces this way are automatically added to your Organization as an Organization Member.
```

File: astro/add-user.md (hunk 1 above)
Comment (jwitz) [inline astro/add-user.md:13]:
```suggestion
- To add edit, or remove Workspace users, you need Workspace Admin permissions for a given Workspace. To add users to a Workspace that are not currently part of your Organization, you need Organization Owner permissions. 
```

File: astro/add-user.md (hunk 2 above)
Comment (jwitz) [inline astro/add-user.md:56]:
```suggestion
4. Select the user's name and email address in the **Organization Member** list, or enter the email for a user hasn't yet been added to your Organization. If you have Organization Owner permissions and invite a new user to a Workspace using their email, that user is also invited to the Organization as an Organization Member. 
```

**Pull request merged**